tests/lean/run/timeFormats.lean

```diff
@@ -801,7 +801,7 @@ Truncation Test
 
 /--
```

Comment:
You should definitely add more tests for the changed behavior.

Reply:
Added more tests.